lib/linter-rust.coffee

```diff
@@ -17,6 +17,9 @@ class LinterRust
       results = []
       file = @initCmd do textEditor.getPath
       curDir = path.dirname file
+      PATH = path.dirname @cmd[0]
+      options = @makeOpts(PATH)
+      options.cwd = curDir
       @cmd.push file
       command = @cmd[0]
       options = {cwd: curDir}
```

Comment:
@gmist: yes you are correct. I pushed a fix again. Sorry. To answer your question about the switch statement. I guess this was just a habit. I once faced an issue with using PATH vs Path on win32 platforms. But after double checking again just now it doesn't seem to be an issue.